src/main/java/study/racingcar/domain/Car.java

```diff
@@ -0,0 +1,40 @@
+package study.racingcar.domain;
+
+public class Car {
+    private String name;
+    private Position position;
+
+    public Car(String name) {
+        this(name, 0);
+    }
+
+    public Car(String name, int position) {
+        this.name = name;
+        this.position = new Position(position);
```

Comment:
아래 코멘트에 대해 고민해 봐요.
https://github.com/next-step/java-racingcar/pull/621/files#r399757690

```suggestion
        this(name, new Position(position));
```